x-pack/plugins/security_solution/public/management/components/endpoint_console/endpoint_response_actions_console_commands.ts

```diff
@@ -34,6 +35,26 @@ export const getEndpointResponseActionsConsoleCommands = (
         },
       },
     },
+    {
+      name: 'release',
+      about: i18n.translate('xpack.securitySolution.endpointConsoleCommands.release.about', {
+        defaultMessage: 'Release the host',
+      }),
+      RenderComponent: ReleaseActionResult,
+      meta: {
+        endpointId: endpointAgentId,
+      },
+      args: {
+        comment: {
+          required: false,
+          allowMultiples: false,
+          about: i18n.translate(
+            'xpack.securitySolution.endpointConsoleCommands.release.arg.command',
```

Comment:
`.command` in the key here should probably be `.comment` since that is the argument's name.

Also - can you make this change as well to the `isolate` command above (looks like I messed that one up too)